### PR TITLE
Fingerprint Cards can go in Folders

### PR DESCRIPTION
**Pull request by Conspiir**
Lets you put those fingerprint sample cards in your case folders. Organization +10.

At the request of SeniorScore.

### CODE REVIEW

**Comment by Conspiir** [comment]
!review

File: code/modules/paperwork/folders.dm [hunk 1]
```diff
@@ -55,6 +55,8 @@
 		dat += "<A href='?src=\ref[src];remove=\ref[Ph]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Ph]'>Rename</A> - <A href='?src=\ref[src];look=\ref[Ph]'>[Ph.name]</A><BR>"
 	for(var/obj/item/weapon/paper_bundle/Pb in src)
 		dat += "<A href='?src=\ref[src];remove=\ref[Pb]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Pb]'>Rename</A> - <A href='?src=\ref[src];browse=\ref[Pb]'>[Pb.name]</A><BR>"
+	for(var/obj/item/weapon/sample/print/Pf in src)
+		dat += "<A href='?src=\ref[src];remove=\ref[Pf]'>Remove</A> <A href='?src=\ref[src];rename=\ref[Pf]'>Rename</A> - <A>[Pf.name]</A><BR>"
```
Comment (Arrow768) [inline code/modules/paperwork/folders.dm]: Why did you add a button to rename it, if you cant rename it.
Remove the rename button here.

File: code/modules/paperwork/folders.dm [hunk 2]
```diff
@@ -107,6 +109,11 @@
 					var/obj/item/weapon/paper_bundle/to_rename = O
 					to_rename.rename()
 
+				else if (istype(O, /obj/item/weapon/sample/print))
+					var/obj/item/weapon/sample/print/F = O
+					usr << "<span class='notice'>You can't rename the [F.name].</span>"
+					return
```
Comment (Arrow768) [inline code/modules/paperwork/folders.dm]: Remove that return here, but keep the rest.

File: code/modules/paperwork/folders.dm [hunk 3]
```diff
@@ -107,6 +109,11 @@
 					var/obj/item/weapon/paper_bundle/to_rename = O
 					to_rename.rename()
 
+				else if (istype(O, /obj/item/weapon/sample/print))
+					var/obj/item/weapon/sample/print/F = O
+					usr << "<span class='notice'>You can't rename the [F.name].</span>"
```
Comment (Arrow768) [inline code/modules/paperwork/folders.dm]: Use to_chat()

File: code/modules/paperwork/folders.dm (hunk 1 above)
Comment (Conspiir) [reply code/modules/paperwork/folders.dm]: The original plan was to keep the cards in-line with the things already in the folder. They had rename buttons, so to keep the fingerprint card from being slid over a full word, I'd keep the button even though the cards named themselves. 

Then I actually started sampling things. Apparently the names can get quite long.

Taking out the rename button is the better play, visually.
